generators/csharp/codegen/src/ast/core/Writer.ts

```diff
@@ -204,6 +204,15 @@ export class Writer {
         return this.buffer;
     }
 
+    public importsToString(): string | undefined {
+        const imports = this.stringifyImports();
+        return imports.length > 0 ? imports : undefined;
+    }
+
+    public getBuffer(): string {
+        return this.buffer;
+    }
```

Comment:
Let's just make the `buffer` property public instead - @dsinghvi recently made a comment in a separate PR, so we should be consistent.

That way we can just access `writer.buffer` instead of `writer.getBuffer()`.

Reply:
👍 will do